### DISCUSSION TITLE
Support GitHub repos that only allow checkout through web URLs

### DISCUSSION THREAD

**Discussion by dennisknothetelus** [oracle/opengrok#4561] [General]
**Is your feature request related to a problem? Please describe.**
My organization uses GitHub enterprise that requires SSO with our corporate IdP, which requires MFA. The organization has disabled repo access via SSH or Personal Access Tokens. Only checkout via web URLs (HTTPS) is allowed. I installed OpenGrok through the default container image, but was not able to configure OpenGrok to access such repositories.

**Describe the solution you'd like**
Extend the OpenGrok GitHub integration to allow access to repos that require checkout via web URLs. Provide detailed instructions how to install OpenGrok through a container image to access such repos.

**Comment by vladak** [comment]
OpenGrok has no Github integration. There is just a plain Git support. Also, the initial clone has to be done has to be done by hand. The tooling present in the Docker image will then resync the repository. It works with HTTP(S) just fine.
